config/db.js

```diff
@@ -1,7 +1,9 @@
 require('dotenv').config();
 const mongoose = require('mongoose');
+const azureService = require('../utils/azureService');
+
+const db = azureService.getSecret('MONGO_URL');
```

Comment:
Rename `azureService` to `configService` because it can take configuration values from multiple places, not from azure only

Reply:
fixed